libs/search-widget/src/components/answer/Chat.scss

```diff
@@ -51,6 +51,9 @@
       grid-template-columns: var(--rhythm-10) auto var(--rhythm-10);
       margin: auto 0 var(--rhythm-4) 0;
     }
+    .answer {
+      width: calc(var(--resource-modal-width-md, fit-content) - var(--rhythm-10) - var(--rhythm-16));
```

Comment:
this is super weird: `fit-content` is there for to fit the content of the div, so substracting a value to it would probably make the width too small… Also, the variable `--resource-modal-width-md` is meant to be used on modals, but as far as I know the chat is like a tile, so it is fullscreen when expanded, no?